tests/phpunit/tests/Story_Renderer/HTML.php

```diff
@@ -125,7 +125,7 @@ public function test_replace_html_head_invalid() {
 	 */
 	public function test_add_publisher_logo() {
 		$attachment_id = self::factory()->attachment->create_upload_object( __DIR__ . '/../../data/attachment.jpg', 0 );
-		add_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO, $attachment_id );
+		update_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO, $attachment_id );
```

Comment:
Why was this change needed? Sounds like we're not properly cleaning that up somewhere else.

Reply:
I don't know why this change is needed, it seems unrelated. I will dig into why.